lang/LangSource/PyrObject.cpp

```diff
@@ -110,7 +110,7 @@ PyrSymbol *s_identitydictionary;
 PyrSymbol *s_linkedlist;
 PyrSymbol *s_sortedlist;
 PyrSymbol *s_array;
-PyrSymbol *s_list, *s_method, *s_fundef, *s_frame, *s_class;
+PyrSymbol *s_list, *s_rest, *s_method, *s_fundef, *s_frame, *s_class;
```

Comment:
It's a fair assumption! Especially with the lack of comments in that code. I guess I'm a little touchy about this instance in particular because I [put through a PR awhile ago](https://github.com/supercollider/supercollider/pull/2627) trying to clean up some of the mess in `PyrObject`.